src/app/workspace/resource/resource-instance-form/resource-instance-form.component.ts

```diff
@@ -285,6 +304,11 @@ export class ResourceInstanceFormComponent implements OnInit, OnDestroy {
                                 // since the component already exists, the ngAfterInit method of the component will not be called so we must assign the value here manually
                                 this.selectResourceClassComponent.form.controls.resources.setValue(this.resourceClasses[0].id);
                             }
+
+                            // notifies the user that the select ontology has got 0 resource class yet defined.
```

Comment:
notifies the user that the selected ontology does not have any resource classes defined yet.